app/assets/javascripts/modules/moj.checkbox-summary.js

```diff
@@ -10,14 +12,20 @@
 
   CheckboxSummary.prototype = {
     init: function ($el, options) {
+      var sum;
       this.cacheEls($el);
       this.bindEvents();
-      this.settings = $.extend({}, {
-        glue: ', ',
-        strip: ';',
-        sub: ' '
-      }, options);
-      this.settings.original = this.$summaries.first().text() || '[summary]';
+      this.settings = $.extend({}, this.defaults, options);
+      if (sum = this.$summaries.first().text()) {
+        this.settings.original = sum;
+      }
+    },
+
+    defaults: {
+      glue: ', ',
+      strip: ';',
+      sub: ' ',
+      original: '[summary]'
     },
```

Comment:
Can we come up with some better defaults for this? Replacing `;` with ` ` by default seems a bit weird to me, and it might be nice to have a nicer value for `original`.